src/app/components/dashboard-apps-dock/no-apps-notification.html

```diff
@@ -0,0 +1,6 @@
+<div>
+  <div class="close top-right-corner" type="button" aria-label="Close" ng-click='toggle()'>x</div>
+  <strong><i class='fa fa-info-circle' aria-hidden='true'></i></strong>
+  {{ 'mno_enterprise.templates.impac.dock.do_not_have_apps' | translate }} 
+  <a ui-sref='onboarding.step2'>{{ 'mno_enterprise.templates.impac.dock.click_me' | translate }} </a> {{ 'mno_enterprise.templates.impac.dock.see_onboarding' | translate }}
```

Comment:
Hi @alexnoox , sorry for invonvenience.
I do not see how I could make this all in one single locale, since the "click here" must be separate. Also, I do not see why I would need to change the ui-sref since it works with this locale. I have pushed one new commit with some changes and improvements along.
Can you please check these particular lines and comment it?
https://github.com/maestrano/mno-enterprise-angular/pull/247/commits/cfb7019ad3ddc17babef8db9e77ba7f1651aeb11#diff-acc1fbceb3cfa713b815410f9624febdR4

Thanks a lot